### PR TITLE
Code refactor

### PR DESCRIPTION
**Pull request by bprashan**
- Added new input argument 'runtime_args'
- refactor attestation.yml file

Signed-off-by: B, Prashanth Natraj <Prashanth.Natraj.B@intel.com>

### CODE REVIEW

File: src/libs/curated_app_libs.py [hunk 1]
```diff
@@ -199,7 +197,6 @@ def run_test(test_instance, test_yaml_file):
         workload_name = get_workload_name(test_config_dict['docker_image'])
         curation_output = generate_curated_image(test_config_dict, run_with_test_option)
         if "expected_output_infile" in test_config_dict.keys():
-            cleanup_after_test(workload_name)
```
Comment (jinengandhi-intel) [inline src/libs/curated_app_libs.py:202]: Why was this removed?

File: src/libs/curated_app_libs.py [hunk 2]
```diff
@@ -208,7 +205,6 @@ def run_test(test_instance, test_yaml_file):
                 return result
         
         if "expected_output_console" in test_config_dict.keys():
-            cleanup_after_test(workload_name)
```
Comment (jinengandhi-intel) [inline src/libs/curated_app_libs.py:211]: and this too?

File: src/libs/curated_app_libs.py [hunk 3]
```diff
@@ -65,14 +67,12 @@ def pre_actions(test_config_dict):
...
     if os.path.isdir(CURATED_APPS_PATH + "/test_config"):
         shutil.rmtree(CURATED_APPS_PATH + "/test_config")
-    else:
-        os.mkdir(CURATED_APPS_PATH+"/test_config")
-    utils.run_subprocess("cp -rf test_config/* {}".format(CURATED_APPS_PATH+"/test_config"))
+    shutil.copytree("test_config", CURATED_APPS_PATH + "/test_config")
 
     pre_actions_for_verifier_image(test_config_dict, end_key)
 
-    input_ord_list = ['signing_key_path', 'runtime_args', 'runtime_variables', 'runtime_variable_list', 'attestation',
```
Comment (jinengandhi-intel) [inline src/libs/curated_app_libs.py:74]: don't we need the ssl_path also?

File: src/libs/curated_app_libs.py (hunk 1 above)
Comment (bprashan) [reply src/libs/curated_app_libs.py:202]: this will be taken care in finally statement...

File: src/libs/curated_app_libs.py (hunk 3 above)
Comment (bprashan) [reply src/libs/curated_app_libs.py:74]: curated app wont ask for ssl_path.. User has to place the crt files in ssl folder manually.

File: src/libs/curated_app_libs.py (hunk 2 above)
Comment (bprashan) [reply src/libs/curated_app_libs.py:211]: this will be taken care in finally statement...

File: src/libs/curated_app_libs.py (hunk 1 above)
Comment (anjalirai-intel) [reply src/libs/curated_app_libs.py:202]: No, it will not be able to... because there is return before that

File: src/libs/curated_app_libs.py (hunk 2 above)
Comment (anjalirai-intel) [reply src/libs/curated_app_libs.py:211]: same here also.

**Review by anjalirai-intel — APPROVED** [review]

File: src/libs/curated_app_libs.py (hunk 2 above)
Comment (jinengandhi-intel) [reply src/libs/curated_app_libs.py:211]: what did we decide? Even if thought it will return before hitting finally, does it run finally and then return?

Comment (bprashan) [reply src/libs/curated_app_libs.py:211]: finally block will be executed even after return statement execution

**Review by jinengandhi-intel — APPROVED** [review]
